modules/csharp/partials/release-notes/couchbase-mobile-csharp-release-note.3.2.0.adoc

```diff
@@ -1,25 +1,42 @@
 [#maint-3-2-0]
-== 3.2.0 -- TODO 2024
```

Comment:
Yeah, sorry, think we need to remove these sections, as there is no 3.2 release for these platforms